.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+This Pull Request includes a
```

Comment:
Instead of this, let's include the tasks that @balloobbot posts but with checkboxes.

Related issue (if applicable):
Example entry for `configuration.yaml` (if applicable):
```yaml

```